tests/utilities_task_def.go

```diff
@@ -51,11 +51,11 @@ func testTaskDef(t *testing.T, variant string) {
+		expectedContainerDefinitions = fmt.Sprintf("[{\"dependsOn\":[{\"condition\":\"HEALTHY\",\"containerName\":\"envoy\"}],\"environment\":[{\"name\":\"SSM_PATH\",\"value\":\"/sharedtools/%s/\"}],\"essential\":true,\"image\":\"nginx:alpine\",\"logConfiguration\":{\"logDriver\":\"awslogs\",\"options\":{\"awslogs-group\":\"%s\",\"awslogs-region\":\"%s\",\"awslogs-stream-prefix\":\"app\"}},\"name\":\"app\",\"portMappings\":[{\"containerPort\":80}]},{\"environment\":[{\"name\":\"APPMESH_RESOURCE_ARN\",\"value\":\"arn:aws:appmesh:%s:%s:mesh/mesh_name/virtualNode/virtual_node\"},{\"name\":\"ENABLE_ENVOY_XRAY_TRACING\",\"value\":\"1\"}],\"essential\":true,\"healthCheck\":{\"command\":[\"CMD-SHELL\",\"curl -s http://localhost:9901/server_info | grep state | grep -q LIVE\"],\"interval\":5,\"retries\":3,\"startPeriod\":10,\"timeout\":2},\"image\":\"public.ecr.aws/appmesh/aws-appmesh-envoy:v1.23.1.0-prod\",\"logConfiguration\":{\"logDriver\":\"awslogs\",\"options\":{\"awslogs-group\":\"%s\",\"awslogs-region\":\"%s\",\"awslogs-stream-prefix\":\"proxy\"}},\"memory\":500,\"name\":\"envoy\",\"user\":\"1337\"},{\"cpu\":32,\"essential\":true,\"image\":\"amazon/aws-xray-daemon\",\"logConfiguration\":{\"logDriver\":\"awslogs\",\"options\":{\"awslogs-group\":\"%s\",\"awslogs-region\":\"%s\",\"awslogs-stream-prefix\":\"xray\"}},\"memoryReservation\":256,\"name\":\"xray-daemon\",\"portMappings\":[{\"containerPort\":2000,\"protocol\":\"udp\"}],\"user\":\"1337\"}]", expectedName, logGroupName, region, region, accountID, logGroupName, region, logGroupName, region)
```

Comment:
I have scrolled horizontally on these 2 lines at least 3 times now and still have no idea what has changed here LOL. It's gonna be ugly regardless with all the escaped characters, but is there any way to make these lines shorter, or do string concatenation elsewhere?

Reply:
ahhh derp I see it now, public.ecr.aws/appmesh/aws-appmesh-envoy:v1.18.3.0-prod => public.ecr.aws/appmesh/aws-appmesh-envoy:v1.23.1.0-prod

Kinda surprised github didn't highlight that.

Reply:
I'll look into it.